monkestation/code/modules/clothing/shoes/magboots.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/clothing/shoes/magboots/boomboots
+
+	desc = "The ultimate in clown shoe technology."
+	name = "boom boots"
+	icon_state = "boomboot0"
+	item_state = "boomboot0"
+	magboot_state = "boomboot"
+	slowdown = SHOES_SLOWDOWN+1
```

Comment:
SHOES_SLOWDOWN seems to just be 0, so you could make slowdown = 1 instead

Reply:
eh, having it like this helps with readability. Thats the whole point there is a shoes_slowndown and a TRUE and FALSE define anyways